### ISSUE TITLE
[MNG-7137] maven core-its fails on empty repo

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#8304]
**[Slawomir Jaranowski](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slawekjaranowski)** opened **[MNG-7137](https://issues.apache.org/jira/browse/MNG-7137?redirect=false)** and commented

When we have fresh checkout of `apache/maven-integration-testing` and run test by

```java
MAVENCODEBASE=../maven ./run-its.sh
```

test `mng7045 DropUselessAndOutdatedCdiApi.ShouldNotLeakCdiApi()`
fail with:

```java
[ERROR] Failed to execute goal org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0:execute (run) on project test: Execution run of goal org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0:execute failed: Plugin org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0 or one of its dependencies could not be resolved: Could not find artifact org.apache.ant:ant:jar:1.10.8 in central (file:target/null)
```

 
Probable cause is the commit: https://github.com/apache/maven-integration-testing/commit/39a2071ffc68e6f2908609fc8deef54bb30f5f39

After it dependencies of `org.codehaus.gmavenplus:gmavenplus-plugin` is not resolved in proper way.

org.apache.groovy:groovy-ant:4.0.0-alpha-2 has dependency to `org.apache.ant:ant:jar:1.10.9` and version of `org.apache.ant:ant:jar:1.10.8` is skipped.

---
No further details from [MNG-7137](https://issues.apache.org/jira/browse/MNG-7137?redirect=false)

**Comment by jira-importer** [comment]
**[Slawomir Jaranowski](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slawekjaranowski)** commented

Probably we should resolve plugins dependency by `maven-dependency-plugin:resolve-plugins`

Move

```
<dependency>
<groupId>org.codehaus.gmavenplus</groupId>
<artifactId>gmavenplus-plugin</artifactId>
<version>1.11.0</version>
</dependency>
```

from `bootstrap/group-11/pom.xml` to `bootstrap/pom.xml`

as

```
<plugin>
<groupId>org.codehaus.gmavenplus</groupId>
<artifactId>gmavenplus-plugin</artifactId>
<version>1.11.0</version>
</plugin>
```

resolve this issue.

Maybe another maven plugins from bootstrap/* should be moved from `dependency` to `plugin` tags and should be resolve by  `maven-dependency-plugin:resolve-plugins`

**Comment by jira-importer** [comment]
**[Maarten Mulders](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mthmulders)** commented

Fixed in [04203bf2](https://github.com/apache/maven-integration-testing/commit/04203bf232e0dffe3adffa6c1dedb94906094bd8).

**Comment by jira-importer** [comment]
**[Slawomir Jaranowski](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slawekjaranowski)** commented

Ok, I confirm now it is ok
